packages/sdk/src/marketplaces/Trader.ts

```diff
@@ -32,6 +32,19 @@ export interface _TraderConfig extends TraderConfig {
   signer: Signer;
 }
 
+export const traderSupportedChainIds = Object.keys(SupportedChainIdsV4).reduce(
```

Comment:
`SupportedChainIdsV4` is the only reliable source of truth for nft-swap-sdk, and it's an Enum type. Enum when converted to javascript object becomes this weird thing, where keys mapped to values, and values mapped to keys within same object. So `enum Supported { 1: 'eth', 137: 'polygon' }` becomes `{ 1: 'eth', 137: 'polygon', eth: 1, polygon: 137 }`. So that's why I have following code below that extracts only integer keys from that enum, since we need chainIds array in the end.

Reply:
Updated